src/models/reviewDao.js

```diff
@@ -0,0 +1,34 @@
+const database = require("./dataSource");
+
+const createReview = async (id, storeId, rate, text) => {
+  const result = await database.query(
+    `
+    INSERT INTO reviews (
+      user_id, 
+      store_id, 
+      text, 
+      rate
+    ) VALUES (1, ?, ?, ?)`,
+    [storeId, text, rate]
+  );
+
+  return result.insertId;
+};
+
+const createReviewImage = async (rowImage) => {
+  const result = await database.query(
+    `
+      INSERT INTO review_images (
+        review_id,
+        image
+      ) VALUES ?
+      `,
+    [rowImage]
+  );
+  return result;
+};
```

Comment:
하나의 transaction으로 처리될 수 있도록 구현해주면 좋을 것 같습니다.

`dataSource.createQueryRunner()` 객체를 활용해서 transaction을 구현해주세요.

```javascript
// create a new query runner
const queryRunner = dataSource.createQueryRunner()

// establish real database connection using our new query runner
await queryRunner.connect()

// now we can execute any queries on a query runner, for example:
await queryRunner.query("SELECT * FROM users")

// we can also access entity manager that works with connection created by a query runner:
const users = await queryRunner.manager.find(User)

// lets now open a new transaction:
await queryRunner.startTransaction()

try {
    // execute some operations on this transaction:
    await queryRunner.manager.save(user1)
    await queryRunner.manager.save(user2)
    await queryRunner.manager.save(photos)

    // commit transaction now:
    await queryRunner.commitTransaction()
} catch (err) {
    // since we have errors let's rollback changes we made
    await queryRunner.rollbackTransaction()
} finally {
    // you need to release query runner which is manually created:
    await queryRunner.release()
}
```